Data/symbol-properties/symbol-properties-database.csv

```diff
@@ -84,6 +84,9 @@ usa,[*],option,,USD,100,0.01,1
 
 usa,[*],equity,,USD,1,0.01,1
 
+usa,[*],index,,USD,1,0.01,1
+usa,[*],indexoption,,USD,100,0.05,1
```

Comment:
Are these properties valid for the most common cases?

Reply:
Yes, the index options we're adding all have the same contract specs